app/controllers/internal/articles_controller.rb

```diff
@@ -39,6 +39,7 @@ def update
     article.update!(article_params)
     Article.where.not(id: article.id).where(live_now: true).update_all(live_now: false) if article.live_now
     CacheBuster.bust("/live_articles")
+    notify(:moderator, current_user, __method__) { cleanse_for_audit(params) }
```

Comment:
Just a suggestion, but I think it may be to easy to forget `cleanse_for_audit` if we have to explicitly call it like that. Maybe just call it from within `notify`?

Reply:
BTW, if you don't always want to pass in `__method__`, `caller_locations(1,1)[0].label` will give you the name of the calling method inside `notify`.